docs/SRE_request_studio_content_api_addition

```diff
@@ -0,0 +1,152 @@
+  - edX-specific learnings from that activity follow here and are incorporated into this document's [runbook](#runbook) section
+
+## Last "go live" date from repo updates was September 20, 2021
+- Review of __api_manager__ repo [commits](https://github.com/edx/api-manager/commits/master)  shows a steady stream of python requirement upgrades, marked as "Verified" commits
+  - The non-verified commits are by the `edx-requirements-bot`
+  - The verified commits are merge commits, frequently run by Usama Sadiq and M Umar Khan, both of Arbisoft
+- Review of the [deployment history for the prod_red stage for the prod environment custom domain name on the API Gateway (prod_black)](https://us-east-1.console.aws.amazon.com/apigateway/home?region=us-east-1#/apis/v96bizdp9b/stages/prod_red) reveals that it is this stage that the python upgrade merges are being applied to as new deployments
+- But the [live stage for the prod environment custom domain name](https://us-east-1.console.aws.amazon.com/apigateway/main/publish/domain-names/api-mappings?domain=api.edx.org&region=us-east-1) is not `prod_red`, but `prod_black`
+  - And `prod_black` has not been updated with new commits since September 20, 2021: no changes to the repo have gone live since that date
+- Similar review of "go live dates" for other stages for the prod environment custom domain are even older
+  - June 27, 2016 for "black"
+  - June 27, 2016 for "red"
+- For the stage environment
+  - The "Yellow" stage is live and is current with the latest repo commits and deployments
+  - Other named stages are behind Yellow, but not by much; unclear who is using these, or how
+
+## The API Gateway supports an "omnibus" API
+- The "API" offered by the API Gateway is an amalgamation of edX APIs (discovery, enterprise, registrar)
+  - Specification of the omnibus API is in [swagger/api.yaml](https://github.com/edx/api-manager/blob/master/swagger/api.yaml)
+- The last [substantive PR](https://github.com/edx/api-manager/pull/127) was on September 20, 2021, adding [v3 enterprise customer endpoints](https://github.com/edx/api-manager/pull/127/files)
+  - That PR and merge line up with the last "go live" date for the prod environment API Gateway (`prod_black` stage)
+
+## Changes made with introduciton of a new API
+The registrar API was added to `api-manager` in commits dated 4/17/2019 through the merge #87 commit of 6/26/2019.
+
+They entail adding the proxy description of the API to `swagger/api.yaml`
+They entail adding a registrar host variable to `scripts/aws/deploy.py`
+
+Corresponding steps were taken to add the Studio content API
+
+## Command line arguments to `deploy.py` and `flip.py`
+
+Both these repo commands take multiple command-line arguments, yet the values supplied with deployment and "go-live"
+events are not documented in the repo.
+
+### Command line arguments for `deploy.py`
+
+The call to the boto/apigateway function that carries out deployments is as follows:
```

Comment:
What's `boto`?

Reply:
The API for programmatically configuring AWS configuration objects